README.md

```diff
@@ -74,14 +74,19 @@ ardrive upload-file --wallet-file /path/to/my/wallet.json --parent-folder-id "f0
     2. [Data Portability](#data-portability)
     3. [Intended Audience](#intended-audience)
 2. [Getting Started](#getting-started)
-    1. [Install Yarn 2 (CLI Users and Developers)](#yarn2)
```

Comment:
Removed most of these "role based" clues in parens. It wasn't obvious how to track through them as a reader.